src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -584,4 +612,3 @@ public boolean isPresent()
         return System.getProperties().containsKey(key);
     }
 }
-
```

Comment:
@jacek-lewandowski this should be probably undeleted.

Reply:
I'll make sure there is no such changes before committing